node/src/gql/types/config.ts

```diff
@@ -22,6 +22,7 @@ export const ConfigType = new GraphQLObjectType({
     urlClassifier: { type: GraphQLString },
     urlGraphql: { type: GraphQLString },
     urlVideo: { type: GraphQLString },
+    idleTipsVideoUrl: { type: GraphQLString },
```

Comment:
can go diff ways, but let's follow the name convention in place here so make it `urlVideoIdleTips`